CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
+### Fixed
+- Alerts open with correct active tab in other supported languages besides English (ie French). [#235]
```

Comment:
Could you add the link to the issue as well?

Reply:
I added the link!

Reply:
Thanks.